code/src/Components/ThoughtsForm.css

```diff
@@ -0,0 +1,90 @@
+
+.message-container {
+    min-width: 50%;
+    max-width: 50%;
+}
+
+.form-message-footer {
+    flex-direction: column;
+    align-items: center;
+}
+
+.form-container {
+    background-color: #f2f0f0;
+}
+
+.form-container h3 {
+    font-family: 'Montserrat', sans-serif;
+}
+
+.text-box {
+    overflow: auto;
+    resize: none;
+    overflow: auto;
+    min-width: 30%;
+    width: 90%;
+    margin-bottom: 10px;
+    font-weight: 500;
+    font-size: 16px;
+    font-family: 'Inconsolata', monospace;
+}
+
+.text-box::placeholder {
+    padding: 5px;
+    color:#f04242;
+    font-weight: 500;
+    font-size: 13px;
+}
+
+p {
+    font-family: 'Montserrat', sans-serif;
+}
+
+.form-button {
+    background: #ffadad;
+    border-radius: 30px;
+    padding: 15px 10px;
+    cursor: pointer;
+    font-size: 15px;
+    font-weight: 600;
+    border: none;
+    /* width: 70%; */
+}
+
+.letter-counter {
+    color: #8d8c8c;
+    font-weight: 600;
+    font-size: 16px; 
+    margin-left: 20px;
+}
+
+@media (min-width: 436px) {
+    .form-button {
+        width: 100%;
+    }
+}
+    
+@media (min-width: 600px) { 
+    
+    .message-container {
+        min-width: 30%;
+        max-width: 30%;
+    }
+    
+    .form-message-footer {
+        display: flex;
+        flex-direction: row;
```

Comment:
When it is in desktop view the "like button" and the "minuts ago" are in a row like you write it out here, but in the phone view it's staked on top of each other. It can be a conscious choice, but if not you can use @media for it;)